themes/govcms/govcms_ui_kit/template.php

```diff
@@ -202,6 +202,13 @@ function govcms_ui_kit_form_alter(&$form, &$form_state, $form_id) {
     // Search form on page not found (404 page).
     $form['basic']['keys']['#title'] = t('Type search term here');
   }
+
+  if (strpos($form_id, 'webform_client_form') !== FALSE) {
+    $form['submitted']['asda'] = array(
```

Comment:
this is embarrassing, I was typing random string for test, but forgot to change :(
it is fixed now